web/packages/shared/components/Notification/Notification.story.tsx

```diff
@@ -36,7 +36,7 @@ export const Notifications = () => {
       <Flex flexDirection="column" gap={4}>
         <Notification
           item={{
-            id: getId(),
+            id: crypto.randomUUID(),
```

Comment:
We have to include the polyfill in `web/.storybook/preview.js` as well. This works when the storybook is executed from within `localhost:9002`, as this is considered to be a secure context. If someone tries to access it over the local IP however, `crypto.randomUUID` won't be available.

```
import '../packages/teleport/src/lib/polyfillRandomUuid';
```